CRM/Core/Smarty.php

```diff
@@ -144,6 +144,7 @@ private function initialize() {
       $this->default_modifiers[] = 'escape:"htmlall"';
     }
     $this->load_filter('pre', 'resetExtScope');
+    $this->load_filter('pre', 'htxtFilter');
```

Comment:
as an aside - from memory this is one thing I am still stuck on with smarty 3 - I think this way of loading filters changes & I haven't figured out what to do instead

Reply:
To my eye, the contract for pre-filters looks pretty similar [v2](https://www.smarty.net/docsv2/en/plugins.prefilters.postfilters.tpl) and [v3](https://www.smarty.net/docs/en/advanced.features.prefilters.tpl). I suppose the name `loadFilter()` is preferred in v3, but it looks like `load_filter()` is still provided (via `SmartyBC`).